fuel-derive/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
```

Comment:
We should also have a README.md for this crate that include status badges for ci and crates.io

Reply:
Added in b16268cb